api/envoy/config/core/v3/grpc_service.proto

```diff
@@ -49,6 +49,10 @@ message GrpcService {
     // Currently only supported for xDS gRPC streams.
     // If not set, xDS gRPC streams default base interval:500ms, maximum interval:30s will be applied.
     RetryPolicy retry_policy = 3;
+
+    // Maximum receive message length.
```

Comment:
```suggestion
    // Maximum gRPC  message size that is allowed to be received. This limit is applied to individual messages in the streaming response and not the total size of streaming response.
    // If a message over this limit is received the gRPC request is terminated with the  RESOURCE_EXHAUSTED error.
```

Reply:
Streaming vs unary response and size limitations are not a client detail. If I'm a developer I need to know how this limit is applied in the unary and streaming gRPC responses. While unary is self-explanatory, since there is only one message, the streaming case is unclear. Does this limit apply to the entire streamed response or just the individual messages within the stream? Clarification would be very helpful.

Reply:
fair point. Thanks! Done.